Ginger/GingerCore/Drivers/JavaDriverLib/GingerJavaAgent/build.xml

```diff
@@ -41,6 +41,8 @@
       <compilerarg value="-XDignore.symbol.file"/>
       <src path="agent"/>
       <!--Uncomment below part only on local eclipse to run test app. On the server it should be always commented-->
+    	<src path="test"/>
```

Comment:
This line should be commented. Only on local when running Test app through eclipse we need it enabled